java/src/main/java/io/simplon/toomanychoco/App.java

```diff
@@ -40,7 +40,7 @@ public void start(int numberOfThreads) throws IOException {
 
 		// Create a basic handler for index route ('/')
 		server.createContext("/", (request -> {
-			String response = "Hello, world!";
+			String response = "bye bye!";
```

Comment:
I think this change overscope your PR
```suggestion
			String response = "Hello, world!";
```